backend/services/implementations/testService.ts

```diff
@@ -66,6 +66,34 @@ class TestService implements ITestService {
       grade: test.grade,
     };
   }
+
+  async getAllTests(): Promise<TestResponseDTO[]> {
```

Comment:
i don't see the method in the interface committed - did you forget to commit it?